py/kubeflow/testing/util.py

```diff
@@ -31,6 +31,12 @@
 # How long to wait in seconds for requests to the ApiServer
 TIMEOUT = 120
 
+DEFAULT_REPOS = {
```

Comment:
These DEFAULTS are really specific to the kfctl_e2e_test.
Lots of E2E tests won't need the tf-operator repo; kfctl only needs it because its running some TFJob tests.

kubeflow/testing is arguably the only repo that would be universally checked out because that's where we are hosting test utilities to be reused across all repos.